<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisTools</class>
 <widget class="QWidget" name="AnalysisTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>79</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
   <item>
    <widget class="QGroupBox" name="CubeGroupBox">
     <property name="title">
      <string>Cube</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
      <item>
       <widget class="QPushButton" name="loadCubeButton">
        <property name="text">
         <string>Load Cube</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bandDownButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/Buttons/Resurces/left_button.png</normaloff>:/Buttons/Resurces/left_button.png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="cubeBandSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>400</number>
        </property>
        <property name="maximum">
         <number>700</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bandUpButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/Buttons/Resurces/right_button.png</normaloff>:/Buttons/Resurces/right_button.png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bandLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showSpectrumButton">
        <property name="text">
         <string>Spectrum</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cubeBandSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bandLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
